rllib/core/models/torch/mlp.py

```diff
@@ -0,0 +1,30 @@
+from ray.rllib.core.models.base import ModelConfig
+from ray.rllib.core.models.base import Model
+from ray.rllib.core.models.torch.primitives import TorchMLP
+from ray.rllib.core.models.torch.base import TorchModel
+from ray.rllib.models.specs.specs_torch import TorchTensorSpec
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.framework import try_import_torch
+
+torch, nn = try_import_torch()
+
+
+class TorchMLPModel(TorchModel, nn.Module):
```

Comment:
So for MLP, the primitive is used in both, MLPEncoder and MLPModel.
For MLPModel and MLPEncoder I think this makes sense especially because it strictly unifies their configs and the logic that builds the layers. MLPModel and MLPEncoder only wrap the primitive model. So if we were to get rid of the primitive MLP, Encoder would have to wrap MLPModel in order to avoid code duplication. Would that be better in your opinion? 

I think having three MLP classes makes sense but admit that it could be clearer in terms of documentation.

Reply:
Going with `TorchMLPHead` here now, because I believe head is more easily and universally understood.